app.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+const { getCategories } = require('./controllers/categories.controller')
+const app = express()
+app.use(express.json());
+
+app.get('/api/categories', getCategories)
+
+app.use((req, res, next) => {
+    res.status(404).send({ msg: "404 not found" });
+});
+app.use((err, req, res, next) => {
+    if (err.status && err.msg) {
+        res.send(err.status).send({msg: err.msg})
+    }
+})
+
+app.use((err, req, res, next) => {
+    if (err.code === '23503') {
+        res.status(404).send({msg: "not found"})
+    }
+})
```

Comment:
are you using these error handling right now in this ticket? If not then please remove